src/driver/op.rs

```diff
@@ -21,6 +21,7 @@ pub(crate) struct Op<T: 'static> {
 }
 
 /// Operation completion. Returns stored state with the result of the operation.
+#[allow(dead_code)]
```

Comment:
What dead code are we allowing?

Reply:
I do not know.

Reply:
Let's not add these sorts of statements then.